BREAKING.md

```diff
@@ -30,6 +31,22 @@ The previous format was just a property bag:
 Whereas now it has nested property bags for error categories including `all` and `error`:
     `ChildLogger.create(logger, undefined, {all:{ someProperty: uuid() }});`
 
+### IContainerRuntimeOptions Format Change
+
+The runtime options passed into `ContainerRuntime` have been subdivided into nested objects, because all of them fall under two categories currently:
+
+-   `summaryOptions` - contains all summary/summarizer related options
+    -   `generateSummaries`
+    -   `initialSummarizerDelayMs`
+    -   `summaryConfigOverrides`
+    -   `disableIsolatedChannels`
+-   `gcOptions` - contains all GC related options
```

Comment:
nit: Consider using `Garbage Collection` instead of `GC` in this sentence. It might not be apparent what GC is.